src/isNegativeZero.js

```diff
@@ -0,0 +1,22 @@
+import { identical } from 'ramda';
+
+/**
+ * Checks if value is a negative zero (-0).
+ *
+ * @func isNegativeZero
+ * @memberof RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.21.0|v2.21.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link https://ramdajs.com/docs/#identical|R.identical}
```

Comment:
We already have a PR for positive zero. Would be good to link to it.
`@see {@link https://ramdajs.com/docs/#identical|R.identical}`

I'd drop identical here, it's more an implementation detail that something that can really help you navigating the library.

Reply:
Agreed!
Done ✅